resources/public/api/conf/3.0/high_income_child_benefit_charge_delete.yaml

```diff
@@ -0,0 +1,125 @@
+summary: Delete High Income Child Benefit Charge Submission
+description: |
+  This endpoint allows a customer to delete the information used to calculate a charge from High Income Child Benefit earners. A National Insurance number and tax year must be provided.
+  
+  ### Test data
+  Scenario simulation using Gov-Test-Scenario headers is only available in the sandbox environment.  
+  
+  | Header Value (Gov-Test-Scenario) | Scenario                                                                                                                     |
+  |----------------------------------|------------------------------------------------------------------------------------------------------------------------------|
+  | N/A - DEFAULT                    | Simulates a success response.                                                                                                |
+  | NOT_FOUND                        | Simulates the scenario where no data is found.                                                                               |
+  | OUTSIDE_AMENDMENT_WINDOW         | Simulates the scenario where request cannot be completed as it is outside the amendment window.                              |
+  | TAX_YEAR_NOT_SUPPORTED           | Simulates the scenario where the tax year is not supported.                                                                  |
+  | REQUEST_CANNOT_BE_FULFILLED      | Simulates the scenario where the request cannot be fulfilled for various reasons other than the ones triggering other errors.|
+  | STATEFUL                         | Performs a stateful delete.                                                                                                  |
+
+parameters:
+  - $ref: './common/pathParameters.yaml#/components/parameters/nino'
+  - $ref: './common/pathParameters.yaml#/components/parameters/taxYear'
```

Comment:
There isn't a component `taxYear` to map to. Looks like the one we will need is `minimumTaxYear2025`